src/index.js

```diff
@@ -25,19 +27,35 @@ async function runTest(files, reporter, requireFilePath) {
             allTestFiles.push(requireFilePath);
         }
         allTestFiles.push(...files);
-        await brs.execute(allTestFiles, {
+        let returnVals = await brs.execute(allTestFiles, {
             stdout: reporterStream,
             stderr: process.stderr
         });
+
         reporterStream.end();
+
+        if (forbidFocused && returnVals.length > 0) {
+            // iterate through return values and see if there are focused cases
+            for (output of returnVals) {
+                let focusedFiles = output.getValue().get("fileswithfocusedcases").getElements();
+                if (focusedFiles.length > 0) {
+                    let formattedList = focusedFiles.map((brsStringPath) => `    ${brsStringPath.value}`).join("/n");
+                    console.error(c.red(`Error: used command line arg ${c.cyan("--forbid-focused")} but found focused tests in these files:`));
+                    console.error(formattedList);
```

Comment:
Just a 💭 ... I wonder if it'd be more appropriate to collapse these into a single `error()` statement for better readability in the Dev Console? 🤷‍♂️ 🐳